js/src/package.js

```diff
@@ -11,7 +11,22 @@ const id = process.env.NODE_ENV == 'production'
   ? location.pathname.split('/').slice(3).join('/')
   : location.search.substring(1);
 
+if (!NodeList.prototype.forEach) {
+  NodeList.prototype.forEach = Array.prototype.forEach;
```

Comment:
Sure. We're extending the prototype for the same already in production.js though